develop/toolkit/api-libraries/dedot.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Dedot
+description: Dedot is a next-generation JavaScript client for Polkadot and Polkadot SDK-based blockchains, offering lightweight, tree-shakable APIs with strong TypeScript support.
```

Comment:
The description is too long. It should be between 120 and 160 characters. This one has 166